src/include/execution/sql/aggregators.h

```diff
@@ -283,6 +285,77 @@ class IntegerMaxAggregate {
   bool null_{true};
 };
 
+/*
+ * TopKAggregate
+ */
+template <typename T>
+class TopKAggregate {
+  using CppType = decltype(T::val_);
+
+ public:
+  /**
+   * Constructor.
+   */
+  explicit TopKAggregate(size_t topK) : histogram_(topK, 64) {}
```

Comment:
Why "64"?

Reply:
Because all sql types have c++ primitives of 64 bytes always.